wallet/src/Pos/Wallet/Web/Methods/Payment.hs

```diff
@@ -137,6 +142,23 @@ instance
     getNewAddress (accId, passphrase) = do
         clientAddress <- L.newAddress RandomSeed passphrase accId
         decodeCTypeOrFail (cadId clientAddress)
+    -- We are creating a sample HD address here. It has the same
+    -- length (76) as the biggest one created by Daedalus.
+    getFakeChangeAddress = do
+        let passphrase = DBA.replicate 10 255
+        SecretKey sk <- snd <$> keyGen
+        encSK <- mkEncSecret passphrase sk
+        case deriveLvl2KeyPair
+                 (IsBootstrapEraAddr True)
+                 (ShouldCheckPassphrase False)
+                 passphrase
+                 encSK
+                 maxBound
+                 maxBound of
+            Nothing -> throwM $ InternalError "getFakeChangeAddress failed"
+            Just (addr, _)
+                | biSize addr == maxHDAddressSizeBoot -> return addr
```

Comment:
dunno why you don't want to use a hardcoded address with the max length, it will look safer. wouldn't you like to check that `biSize addr >= maxHDAddressSizeBoot`?

Reply:
AFAIU it can't be `>` since `maxHDAddressSizeBoot` is exactly what it says -- it's a _maximum_ size.

Reply:
If it's `>`, it means there is a bug. There is an issue to write tests for this function. In particular we want to test that it terminates. If at some point `maxHDAddressSizeBoot` stops being maximum, we will notice it by failing tests. But if it's `>=`, maybe we won't.